### DISCUSSION TITLE
AzurePipelines Scaled - Logs and Typos

### DISCUSSION THREAD

**Discussion by endersonmenezes** [kedacore/keda#2277] [Q&A / Need Help] [answered]
I'm trying to contribute with this repo.

In my team, we added this line to help debug log output.

https://github.com/kedacore/keda/pull/2276

What do you think? I can try to take a look at the log messages and improve them, I believe if they had more information it would help.

**Comment by zroubalik** [comment, marked as answer]
That PR is closed, but we are generally open to any contributions that help improve KEDA :)

**Reply by endersonmenezes** [reply]
Yes, I closed it because I was afraid people would find it boring for me to open the PR.

**Reply by zroubalik** [reply]
Opening a PR is never boring :)

**Reply by JorTurFer** [reply]
Hi @endersonmenezes
I agree with @zroubalik, helping is never boring :)
Your change makes sense totally
